### ISSUE TITLE
IndexError: list index out of range

### ISSUE THREAD

**Issue by jim-minter** [Azure/OpenShift#96]
```
Anjalis-MacBook-Pro:~ atelang$ az openshift create --resource-group testanjone --name testanjone -l eastus --fqdn testanjone.eastus.cloudapp.azure.com
The command failed with an unexpected error. Here is the traceback:

list index out of range

Traceback (most recent call last):
  File "/usr/local/Cellar/azure-cli/2.0.62/libexec/lib/python3.7/site-packages/knack/cli.py", line 206, in invoke
    cmd_result = self.invocation.execute(args)
  File "/usr/local/Cellar/azure-cli/2.0.62/libexec/lib/python3.7/site-packages/azure/cli/core/commands/__init__.py", line 326, in execute
    raise ex
  File "/usr/local/Cellar/azure-cli/2.0.62/libexec/lib/python3.7/site-packages/azure/cli/core/commands/__init__.py", line 384, in _run_jobs_serially
    results.append(self._run_job(expanded_arg, cmd_copy))
  File "/usr/local/Cellar/azure-cli/2.0.62/libexec/lib/python3.7/site-packages/azure/cli/core/commands/__init__.py", line 377, in _run_job
    six.reraise(*sys.exc_info())
  File "/usr/local/Cellar/azure-cli/2.0.62/libexec/lib/python3.7/site-packages/six.py", line 693, in reraise
    raise value
  File "/usr/local/Cellar/azure-cli/2.0.62/libexec/lib/python3.7/site-packages/azure/cli/core/commands/__init__.py", line 354, in _run_job
    result = cmd_copy(params)
  File "/usr/local/Cellar/azure-cli/2.0.62/libexec/lib/python3.7/site-packages/azure/cli/core/commands/__init__.py", line 145, in __call__
    return self.handler(*args, **kwargs)
  File "/usr/local/Cellar/azure-cli/2.0.62/libexec/lib/python3.7/site-packages/azure/cli/core/__init__.py", line 451, in default_command_handler
    return op(**command_args)
  File "/usr/local/Cellar/azure-cli/2.0.62/libexec/lib/python3.7/site-packages/azure/cli/command_modules/acs/custom.py", line 2531, in openshift_create
    name=name, update=update_aad_secret)
  File "/usr/local/Cellar/azure-cli/2.0.62/libexec/lib/python3.7/site-packages/azure/cli/command_modules/acs/custom.py", line 2284, in _ensure_osa_aad
    aad_client_app_id = list_aad_filtered[0].app_id
IndexError: list index out of range
```

**Comment by jim-minter** [comment]
@julienstroheker ptal - this is happening because the AAD account is  testanjone.westus.cloudapp.azure.com but the cluster exists in eastus.

Not sure how that happened - perhaps at some point an `az openshift create -l eastus --fqdn ...westus...`?

**Comment by julienstroheker** [comment]
This should be fix with cli 2.0.64 can you please let me know if this is still happening? Thanks
